examples/unet_segmentation_3d_dict.py

```diff
@@ -52,15 +52,22 @@
 
 images = sorted(glob(os.path.join(tempdir, 'im*.nii.gz')))
 segs = sorted(glob(os.path.join(tempdir, 'seg*.nii.gz')))
+train_data = list()
+for img, seg in zip(images[:30], segs[:30]):
```

Comment:
can be rewritten as 
```suggestion
train_data = [{'img': img, 'seg': seg} for img, seg in zip(images[:30], segs[:30])]
```
more pythonic.

Reply:
Updated in the latest patch.
Thanks.